src/osd/OSD.cc

```diff
@@ -9889,6 +9891,9 @@ void OSD::handle_conf_change(const ConfigProxy& conf,
   if (changed.count("osd_max_trimming_pgs")) {
     service.snap_reserver.set_max(cct->_conf->osd_max_trimming_pgs);
   }
+  if (changed.count("osd_max_scrubs")) {
+    service.scrub_reserver.set_max(cct->_conf->osd_max_scrubs);
```

Comment:
Nit: with this change we effectively allow osd_max_scrubs primary scrubs and osd_max_scrubs replica scrubs, right?  That's reasonable to me, but it's probably worth noting that in the config description.

Reply:
That change was introduced in another PR, on November 2023.
And - yes - I should change the documentation.